scripts/config.js

```diff
@@ -1,4 +1,4 @@
-export const START_YEAR = 2014;
+export const START_YEAR = 2018;
```

Comment:
I now run the tests starting in 2018 otherwise we can't see the logs correctly on TravisCI when there is an error. It was too long.